### PR TITLE
Added 3 resources in 2 different categories

### PR DESCRIPTION
**Pull request by ayushkumarbhadani**
## Changes proposed
Added 3 resources in 2 different categories

1. Computer Science: 
  > 5 Minutes Engineering
  > Jenny's Lectures CS IT

2. Competitive Programming:
  > Luv

### CODE REVIEW

File: database/youtube/computer-science.json [hunk 1]
```diff
@@ -6,5 +6,21 @@
...
+    "url": "https://www.youtube.com/@JennyslecturesCSIT",
+    "category": "youtube",
+    "subcategory": "computer-science",
+    "language": "english"
+  },
+  {
+    "name": "5 Minutes Engineering",
+    "description": "On 5 Minutes Engineering you can find EASIEST explanations for all following mentioned subjects in HINDI: Aptitude, Machine Learning, Deep Learning, C, R and Python Programming Languages, Discrete Mathematics (DM), Theory Of Computation (TOC), Artificial Intelligence(AI), Database Management System(DBMS), Software Modeling and Designing(SMD), Software Engineering and Project Planning(SEPM), Data mining and Warehouse(DMW), Data analytics(DA), Mobile Communication(MC), Computer networks(CN), High performance Computing(HPC), Operating system, System programming (SPOS), Internet of things(IOT), Design and analysis of algorithm(DAA)",
```
Comment (CBID2) [inline database/youtube/computer-science.json:20]: [Proofreading]
```suggestion
    "description": "On 5 Minutes Engineering, you can find EASIEST explanations for all following mentioned subjects in HINDI: Aptitude, Machine Learning, Deep Learning, C, R and Python Programming Languages, Discrete Mathematics (DM), Theory Of Computation (TOC), Artificial Intelligence(AI), Database Management System(DBMS), Software Modeling and Designing(SMD), Software Engineering and Project Planning(SEPM), Data mining and Warehouse(DMW), Data analytics(DA), Mobile Communication(MC), Computer networks(CN), High performance Computing(HPC), Operating system, System programming (SPOS), Internet of things(IOT), and Design and analysis of algorithm(DAA)",
```

File: database/youtube/computer-science.json [hunk 2]
```diff
@@ -6,5 +6,21 @@
     "category": "youtube",
     "subcategory": "computer-science",
     "language": "english"
+  },
+  {
+    "name": "Jenny's Lectures CS IT",
+    "description": "Jenny’s Lectures CS IT is a Free YouTube Channel providing Computer Science / Information Technology / Computer-related tutorials including NET & JRF Coaching Videos, GATE Coaching Videos, UGC NET, NTA NET, JRF, BTech, MTech, Ph.D., tips and other helpful videos for Computer Science / Information Technology students. Teaching Computer Science in Informal Space. ",
```
Comment (CBID2) [inline database/youtube/computer-science.json:12]: [Proofreading]
```suggestion
    "description": "Jenny’s Lectures CS IT is a Free YouTube Channel providing Computer Science / Information Technology / Computer-related tutorials, including NET & JRF Coaching Videos, GATE Coaching Videos, UGC NET, NTA NET, JRF, BTech, MTech, Ph.D., tips and other helpful videos for Computer Science / Information Technology students.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @ayushkumarbhadani! I just made some proofreading suggestions in my inline comments. In the future, try to describe the source in your own words and run the descriptions through Spellcheck or Grammarly before submitting them as pull requests.

**Comment by ayushkumarbhadani** [comment]
Hey @CBID2, Kindly re-review it, please. Hope I didn't make any mistakes this time.

**Review by CBID2 — APPROVED** [review]
Great job @ayushkumarbhadani ! 😄 LGTM! 😄

**Review by rupali-codes — CHANGES_REQUESTED** [review]
hey, could you please add the playlist link instead? because these youtube channel has also been added in different categories

**Comment by ayushkumarbhadani** [comment]
Sure @rupali-codes

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**